pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -598,8 +679,15 @@ func lookupAppInstanceStatus(ctx *zedmanagerContext, key string) *types.AppInsta
 	return &status
 }
 
-func lookupAppInstanceConfig(ctx *zedmanagerContext, key string) *types.AppInstanceConfig {
-
+func lookupAppInstanceConfig(ctx *zedmanagerContext, key string, checkLocal bool) *types.AppInstanceConfig {
```

Comment:
Might make sense to document that is islocal is set we give preference to the local config from a snapshot.

Reply:
Done